source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.importance.get_param_importances.po

```diff
@@ -100,13 +107,14 @@ msgid ""
 "A function to specify the value to evaluate importances. If it is "
 ":obj:`None` and ``study`` is being used for single-objective "
 "optimization, the objective values are used."
-msgstr ""
+msgstr "一个用于指定求解重要性的值的函数. 如果将其设置为 :obj:`None` 并且"
+"``study`` 是一个单目标优化的话, 这个值将会采用目标函数."
```

Comment:
ditto

Reply:
Line 99 should not be translated as we have `  .. note::     ` in that. Besides, translation of line 99 is actually finished by line 110 and 117.